frappe/core/doctype/doctype/doctype.json

```diff
@@ -44,6 +44,7 @@
   "allow_auto_repeat",
```

Comment:
Update `modified` of this file... or else changes will not be replicated while migration